reference/customers.v3.yml

```diff
@@ -338,8 +342,8 @@ paths:
 
 
         **Notes**
-
-        Attributes Values can not be updated using Update a Customer. Use the Update a [Customer Attribute Values](https://developer.bigcommerce.com/api-reference/customer-subscribers/v3-customers-api/customer-attribute-values/customersattributevaluesput) endpoint.
+        * Attributes Values can not be updated using Update a Customer. Use the Update a [Customer Attribute Values](https://developer.bigcommerce.com/api-reference/customer-subscribers/v3-customers-api/customer-attribute-values/customersattributevaluesput) endpoint.
+        * channel_ids -- Updating this list of channels a customer can access to may create some side effects in a multi-storefront situation as this list is used to determine which customer account we will use to authenticate a shopper given a channel.
```

Comment:
I feel everything around `origin_channel_id` and `channel_ids` is quite complex and I would suggest creating an article just for this. Then we could potentially just link the page here.
Otherwise I'm struggling a bit to explain all the logics, I would probably need to add examples and a lot of explanation for this feature to be fully understood.